scripts/image-builder/config/kickstart.ks.template

```diff
@@ -58,5 +58,7 @@ chown -R redhat:redhat /home/redhat/
 
 # Configure the firewall
 firewall-offline-cmd --zone=trusted --add-source=10.42.0.0/16
+firewall-offline-cmd --zone=trusted --add-source=169.254.169.1
+systemctl is-enabled --quiet firewalld.service && firewall-cmd --reload
```

Comment:
This is a setting that probably belongs into the blueprint itself, together with enabling the microshift service. This way it works with both standard and
simplified installers.

Note the firewall rules will eventually need to move into the blueprint, too; this is just a workaround as the composer still doesn't support this type of rules.

Reply:
Yes, it's a temporary setting. We need something to have ISO working, so this seems to be a sensible compromise for now.